src/main/resources/application.yaml

```diff
@@ -25,6 +25,10 @@ stargate:
   exception-mappers:
     enabled: false
 
+  feature:
+    flags:
+      TABLES: null
```

Comment:
`null` meaning "no default", allowing per-request override, whereas `true` would mean enabled and `false` disabled (neither allowing per-request overrides)

Reply:
Actually, better choice is to leave it empty; either works.

Reply:
works for me